openedx/core/djangoapps/credit/tasks.py

```diff
@@ -75,8 +74,15 @@ def _get_course_credit_requirements(course_key):
     credit_xblock_requirements = _get_credit_course_requirement_xblocks(course_key)
     min_grade_requirement = _get_min_grade_requirement(course_key)
     proctored_exams_requirements = _get_proctoring_requirements(course_key)
+    block_requirements = credit_xblock_requirements + proctored_exams_requirements
+    # sort credit requirements list based on start date and put all the
+    # requirements with no start date at the end of requirement list.
```

Comment:
I'm not sure if this is an outdated comment or if the code needs to be updated.  If the start date is `None`, the requirement should be prioritized to the beginning of the list.